docs/Reference.md

```diff
@@ -247,6 +247,40 @@ If `false`, (only supported on iOS and macOS), loads it into all frames (e.g. if
 
 ---
 
+### `injectJavaScriptObject`[⬆](#props-index)<!-- Link generated with jump2header -->
+
+Inject any JavaScript object into the webview so it is available to the JS running on the page.
+
+| Type | Required | Platform                                          |
+| ---- | -------- | ------------------------------------------------- |
+| obj | No       | Android only |
+
+Example:
+
+Set an auth token to be used in JavaScript.
```

Comment:
In general the docs should warn that this object will be accessible to all frames of the web page.... this auth token scenario is a good example of where that might be a security risk for some apps. (and thus _might_ be a bad example...)